dissect/target/loaders/dir.py

```diff
@@ -11,15 +11,24 @@
 if TYPE_CHECKING:
     from dissect.target import Target
 
+PREFIXES = ["", "fs"]
+
+
+def find_entry_path(path: Path) -> str | None:
```

Comment:
```suggestion
def find_entry_path(path: Path) -> Optional[str]:
```
I don't think using `|` in type hints is legal in Python 3.9?

Could you also move this below the `DirLoader` class itself? That way it's more consistent placing with the other functions.